src/webots/nodes/utils/WbSupervisorUtilities.cpp

```diff
@@ -1346,7 +1346,7 @@ void WbSupervisorUtilities::handleMessage(QDataStream &stream) {
       }
 
       if (enable) {
-        WbTransform *const fromTransformNode = fromNode ? dynamic_cast<WbTransform *>(fromNode) : NULL;
+        WbPose *const fromTransformNode = fromNode ? dynamic_cast<WbPose *>(fromNode) : NULL;
         if (fromNodeId && !fromTransformNode)
           mRobot->warn(
             tr("Pose tracking can be exclusively used with Transform (or derived) 'from_node' argument, but '%1' (%2) is "
```

Comment:
It seems the warnings of the get and track pose functions (and the variable names) are not fully converted to the Pose nodes.
```suggestion
            tr("Pose tracking can be exclusively used with Pose (or derived) 'from_node' argument, but '%1' (%2) is "
```